docs/topics/testing/tools.txt

```diff
@@ -1448,7 +1448,6 @@ USE_TZ, TIME_ZONE                Databases timezone
 TEMPLATES                        Template engines
 SERIALIZATION_MODULES            Serializers cache
 LOCALE_PATHS, LANGUAGE_CODE      Default translation and loaded translations
-MEDIA_ROOT, DEFAULT_FILE_STORAGE Default file storage
```

Comment:
I think there needs to be an addition here (even if below this table?) saying that the loaded storages are reset too. 🤔

Reply:
I've updated this table with settings that change storages configs